src/newViews/AnalysisModule/Analysis/PillarList/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useCallback, useState, useMemo } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import {
+    Pager,
+    Container,
+    ListView,
+} from '@the-deep/deep-ui';
+
+import { useRequest, useLazyRequest } from '#utils/request';
+
+import {
+    MultiResponse,
+    AnalysisPillars,
+} from '#typings';
+
+import AnalysisPillar from '../AnalysisPillar';
+
+import styles from './styles.scss';
+
+const MAX_ITEMS_PER_PAGE = 5;
+const keySelector = (item: AnalysisPillars) => (item.id);
+
+interface Props {
+    className?: string;
+    createdAt: string;
+    modifiedAt: string;
+    activeProject: number;
+    onAnalysisPillarDelete: () => void;
+    analysisId: number;
+}
+
+function AnalysisDetails(props: Props) {
+    const {
+        className,
+        createdAt,
+        modifiedAt,
+        activeProject,
+        onAnalysisPillarDelete,
+        analysisId,
+    } = props;
+
+    const [activePage, setActivePage] = useState<number>(1);
```

Comment:
```suggestion
    const [activePage, setActivePage] = useState(1);
```
The typing will be automatically inferred